v1

```diff
@@ -0,0 +1 @@
+{"artist": "The Strokes", "title": "Trying Your Luck", "art_url": "https://i.scdn.co/image/ab67616d0000b27313f2466b83507515291acce4", "album": "Is This It", "state": "Playing", "state_changed_time": 1719504662.7256083, "connected": true}
```

Comment:
what do you call this type of music again?

Reply:
:rofl: